### DISCUSSION TITLE
建议能屏蔽特殊弹幕形式

### DISCUSSION THREAD

**Discussion by strange-qwq** [the1812/Bilibili-Evolved#4773] [功能建议 / Ideas]
比如像视频BV1Vx4y1J7Qh这种斜着发的弹幕，还有拜年祭直播时烟花一样的弹幕，很影响弹幕观感，尝试了`禁用特殊弹幕样式`组件，但是里面并没有这种特殊弹幕形式
![image](https://github.com/the1812/Bilibili-Evolved/assets/73089181/b3c23c6a-762f-4333-95f2-96dc2ffd0788)

![image](https://github.com/the1812/Bilibili-Evolved/assets/73089181/0e8f270a-ed92-4216-8b23-84116f5c5388)

**Comment by the1812** [comment]
![Snipaste_2024-06-03_23-09-49](https://github.com/the1812/Bilibili-Evolved/assets/26504152/e7a8a803-01a6-4e0f-8ab5-eba9f60d746c)

我这咋没有
